source/client-side-encryption/tests/README.md

```diff
@@ -3683,3 +3683,43 @@ Run an aggregate operation on `db.csfle` with the following pipeline:
 ```
 
 Expect an exception to be thrown with a message containing the substring `Upgrade`.
+
+### 26. Custom AWS Credentials
+
+These tests require valid AWS credentials. Refer:
+[Automatic AWS Credentials](../client-side-encryption.md#automatic-credentials).
+
+#### Case 1: Explicit encryption with credentials and custom credential provider
```

Comment:
Do you think we should test this case for auto encryption as well?  Most (all?) drivers have separate code paths for instantiating a CE object and instantiating a client configured for auto encryption.  This test only covers explicit encryption.

Reply:
I've added a case 3 that checks the same for auto encryption.